### PR TITLE
Versions for Images

### PR DESCRIPTION
**Pull request by jiritichy**
Until now, we weren't using the version for images as it wasn't needed. Now, there are quite a lot of changes on the website and it's necessary to make sure we're using the last version.

This PR should introduce this change for every image we're using on the website.

### CODE REVIEW

File: html/clients.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "eScootie",
     "title": "escootie.page.title",
     "link": "/clients/escootie.html",
-    "image": "/images/clients/eScootie-logo-v.1"
```
Comment (michal-simon) [inline html/clients.json:6]: This is definitely not necessary, you can add the version in NJK file

File: html/clients/escootie.njk [hunk 2]
```diff
@@ -6,6 +6,12 @@
 
 {% include 'components/header.njk' %}
 
+<script type="application/ld+json">
+    {
+        "image": "<%= pageUrl %>{{ pageImage }}?v<%= version %>",
```
Comment (michal-simon) [inline html/clients/escootie.njk:11]: I am not sure if just picture will be enough for social media but other than that the versioned image URL is correct.

File: html/components/footer.njk [hunk 3]
```diff
@@ -2,7 +2,7 @@
     <div class="container">
         <div class="row">
             <div class="col-12 col-md-4 pb-4">
-                <img src="/images/logo-v1.svg" alt="{{ t('logo.alt') }}" height="60px">
+                <img src="/images/logo-v1.svg?v<%= version %>" alt="{{ t('logo.alt') }}" height="60px">
```
Comment (michal-simon) [inline html/components/footer.njk:5]: why do you name the picture `logo-v1.svg`?

File: www/images/clients/eScootie-logo-md.jpg
Comment (michal-simon) [inline www/images/clients/eScootie-logo-md.jpg:1]: Why?

File: www/images/clients/eScootie-logo-sm.jpg
Comment (michal-simon) [inline www/images/clients/eScootie-logo-sm.jpg:1]: Why?

**Review by michal-simon — CHANGES_REQUESTED** [review]
The idea is executed mostly well. There are just a few small missteps. I tried to ask you a few questions to steer you into the right direction.

File: html/clients/escootie.njk (hunk 2 above)
Comment (jiritichy) [reply html/clients/escootie.njk:11]: I see, I'll add other attributes for Social medias as well

File: www/images/clients/eScootie-logo-md.jpg
Comment (jiritichy) [reply www/images/clients/eScootie-logo-md.jpg:1]: What do you mean?

File: html/clients/escootie.njk (hunk 2 above)
Comment (jiritichy) [reply html/clients/escootie.njk]: However, I think I'll send another PR for this.

**Review by michal-simon — APPROVED** [review]

**Pull request merged**